parquet-common/src/main/java/org/apache/parquet/Log.java

```diff
@@ -41,6 +34,7 @@
  * @author Julien Le Dem
  *
  */
+@Deprecated
```

Comment:
I'd rather not make this deprecated in this PR since it will make every class have warnings.
Otherwise, this looks good to me.